OxfordBodleian/Juel-Jensen/BDLaethd17.xml

```diff
@@ -527,17 +536,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     </physDesc>
                     <history>
                         <origin>
-                            <origDate notBefore="1650" notAfter="1750" evidence="lettering"> </origDate> The
+                            <origDate notBefore="1650" notAfter="1750" evidence="lettering">mid-seventeenth–mid-eighteenth century</origDate> 
+                            The
                             date of production, taking into account the handwriting and the frame on <locus target="#4r"/>,
-                            is probably a period between the mid-seventeenth to mid-eighteenth century. The name of <persName
+                            is probably a period between the mid-seventeenth to mid-eighteenth century. 
```

Comment:
Above, this type of dates was rather written "mid-seventeenth–mid-eighteenth", I believe. I don't know which is better